src/index.html

```diff
@@ -8,5 +8,40 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <main class="conteiner-calendar">
```

Comment:
What's the purpose of extra container? You can add all the properties to calendar directly